pallets/loans/src/lib.rs

```diff
@@ -849,6 +871,15 @@ pub mod pallet {
 			policy::find_rule(rules, |trigger| loan.check_write_off_trigger(trigger))
 		}
 
+		fn get_change(
```

Comment:
I was wondering initially whether we were properly checking whether the changes were released. This function name doesn't make that clear. I would suggest making this more explicit, something like
```suggestion
		fn get_change_if_released(
```

or `get_released_change` or something else that's better that makes this clear

Reply:
Ok, it's reasonable! Good suggestion